src/Tuura/Boolean/Parser.hs

```diff
@@ -24,17 +24,17 @@ instance Show a => Show (Expr a) where
   show (Var c)     = show c
   show (SubExpr e) = "(" ++ show e ++ ")"
 
-not :: (Expr a) -> (Expr a)
+not :: Expr a -> Expr a
```

Comment:
`not` is used for simply providing inversions in a nicer way. It can most likely be removed, and if it causes an issues, that is in problems in other functions.